axelrod/strategies/titfortat.py

```diff
@@ -92,6 +94,39 @@ class TwoTitsForTat(Player):
     def strategy(opponent: Player) -> Action:
         return D if D in opponent.history[-2:] else C
 
+class DynamicTwoTitsForTat(Player):
+    """A player starts by cooperating and then mimics previous move by 
```

Comment:
This strategy does not mimic the previous move by the opponent. It seeks revenge if the opponent has defected in the previous two rounds.